scss/variables.scss

```
@@ -0,0 +1,97 @@
+/*font-family*/
+$font-family--base: Montserrat,sans-serif;
+//colors//
+$color--white: #ffffff;
+$color--custom--white: rgba(255,255,255,0.85);
+$color--black: #1e1e1e;
+$color--custom-black: #191e2d;
+$color--red: #f0083f;
+$color--dark-green: #86a700;
+$color--gray: #959595;
+$color--custom-gray: #5f5f5f;
+$color--light-gray:#7b7b7b;
+$color--green: rgb(73, 185, 59);
+/*colors-text*/
+$text__color--white: $color--white;
+$text__color--gray: $color--gray;
+$text__color--black: $color--black;
+$text__color--green: $color--dark-green;
+/*font-size-text*/
+$font-size: 12px;
+$font-size--base: 14px;
+$font-size--small: 17px;
+$font-size--medium: 20px;
+$font-size--big: 40px;
+$font-size--huge: 80px;
+/*font-size header*/
+$h1__font-size: 90px;
+$h2__font-size: $font-size--huge;
+$h4__font-size: $font-size--medium;
+/*colors header*/
+$h1__color--white: $color--white;
+$h1__color--dark: $color--black;
+$h2__color-white: $color--white;
+$h2__color-dark: $color--black;
+$h3__color--white: $color--white;
+$h3__color--dark: $color--black;
+$h4__color--white: $color--white;
+$h4__color--dark: $color--black;
+$h5__color--white: $color--white;
+$h5__color--dark: $color--black;
+$h6__color--white: $color--white;
+$h6__color--dark: $color--black;
+/*font-weight*/
+$font-weight--base: 300;
+$text__font-weight: 400;
+$h1__font-weight: 600;
+$h2__font-weight: 400;
+//line-height//
+$line-height--base: 1.75em;
+$h1__line-height: 48px;
+$h2__line-height: 1.2em;
+/*letter-spacing*/
+$letter-spacing: 1px;
+/*background colors*/
+$topbar__background: $color--black;
+$navigation__background: $color--custom--white;
+$canvas__background--dark: $color--black;
+/*hover*/
+$link-hover__background: $color--custom--white;
+$link-hover__background--dark: $color--black;
+$btn__background-hover: $color--green;
```

let's use hover as modifier
```suggestion
$link__background--hover: $color--custom--white;
$link__background--hover-dark: $color--black;
$btn__background--hover: $color--green;
```